app/helpers/funding_helper.rb

```diff
@@ -9,6 +9,11 @@ def scholarship_funding_eligibility(application)
   end
 
   def scholarship_eligibility_in_review?(application)
+    return false if application.eligible_for_funding
+    return false if !application.eligible_for_funding && application.funding_choice.present?
+    return false if application.employment_type == "other"
+    return true if application.teacher_catchment == "england" && application.course.ehco? && new_headteacher?(application)
```

Comment:
Could `application.teacher_catchment == "england"` be separate check like `return false if application.teacher_catchment != "england"` and maybe we could have method like `application.teacher_catchment_in_england?`?

Reply:
Updated